content/docs/guides/ads_on_amp/ads_getting_started.md

```diff
@@ -0,0 +1,63 @@
+---
+$title : "Getting Started"
+$order : 0
+$parent: /content/docs/guides/ads_on_amp
```

Comment:
@yunelee  - were you able to successfully build the doc site with these new docs?  I've cloned your repo and I cannot get it to build.  I get the error "grow.pods.formats.BadFormatError: Unsupported extension for content document: ads_on_amp"